src/UI Component/Login/login.js

```diff
@@ -0,0 +1,121 @@
+
+import React, { useState } from "react";
+import SignUpPage from "./signuppage"
+import AdminDashboard from "../Adminpanel/dashboard";
+import UserDashBoard from "../User/userdashboard";
+
+
+const SignIn = () => {
+  const [formInput, setFormInput] = useState({
+    email: "",
+    password: "",
+  });
+  const [isSignUp, setIsSignUp] = useState(false);
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormInput({ ...formInput, [name]: value });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    // Check if the entered credentials are valid
+    const adminCredentials = {
+      email: "admin@mail.com",
+      password: "password123",
+    };
+
+    const userCredentials = {
+      email: "ephy@mail.com",
+      password: "password123",
+    };
+
+    if (
+      formInput.email === adminCredentials.email &&
+      formInput.password === adminCredentials.password
+    ) {
+      setIsLoggedIn(true);
+    } else if (
+      formInput.email === userCredentials.email &&
+      formInput.password === userCredentials.password
+    ) {
+      setIsLoggedIn(true);
+    } else {
```

Comment:
There is not need for the elseIf here. 

You can easily use a || statement since all you are doing in both conditions is to setIsLoggedIn  to true

```
condition 1 

formInput.email === adminCredentials.email &&
      formInput.password === adminCredentials.password
```

```
condition 2 
formInput.email === userCredentials.email &&
      formInput.password === userCredentials.password

```

if (condition 1 or condition 2)